hadoop-hdfs-project/hadoop-hdfs/src/main/java/io/hops/transaction/context/RootINodeCache.java

```diff
@@ -103,9 +104,9 @@ public Object performTask() throws IOException {
             LOG.debug("RootCache: root does not exist.");
           }
         } catch (InterruptedException e) {
-          e.printStackTrace();
+          LOG.warn(e);
```

Comment:
Shouldn't we also interrupt the current thread if we received an InterruptedException?

Reply:
Yea, I was a little lazy and didn't want to take the risk of changing the behavior of the class, but I will clean that up.